driver-core/src/test/java/com/datastax/driver/core/exceptions/FunctionExecutionExceptionTest.java

```diff
@@ -23,6 +24,7 @@
 import java.util.Collection;
 
 @CassandraVersion(major = 2.2)
+@CCMConfig(config = "enable_user_defined_functions:true")
```

Comment:
correct, the filtering of enable_user_defined_functions is needed for tests that don't require 2.2, but where some of the tests do and require user defined functions.  We could have gotten around it by using PER_METHOD on those particular tests, but it would have needed to create a separate cluster, so this is pretty much just a small optimization.